docs/environment-setup.md

```diff
@@ -137,67 +137,81 @@ _NOTE: This will not work on WSL (Windows Subsystem for Linux). Use the approach
 
 **Important:** Docker builds Telescope's dependencies at launch and keeps them on disk. In some cases, Docker might try to reuse already-built dependencies or cached data, causing misleading results when testing Telescope. To avoid this, it is recommended to use the command `docker system prune -af --volumes` to remove all already-built Telescope dependencies and ensure fresh deployments.
 More information about docker: [images vs containers](https://www.baeldung.com/docker-images-vs-containers) and [volumes](https://docs.docker.com/storage/volumes/).
 
-#### Option 2: Natively installed:
+### Start telescope
 
-#### Redis:
+There are different ways to run the application. By default, `.env` will be read; however, if there is no `.env` file in the root, [env.development](../config/env.development) will be used.
 
-Run `redis-server`
+There are also [env.production](../config/env.production) and [env.staging](../config/env.staging) to choose based on developer's need. For example, if you want to use staging, you can do `cp ./config/env.staging ./.env` on Linux/macOS, or `copy config/env.staging .env` on windows.
 
-_Note: If experiencing an error such as `Error starting userland proxy: listen tcp 0.0.0.0:6379: bind: address already in use` when using either option to start Redis. Check if an existing instance of redis is already running (Docker/Native) and stop it before starting another instance_
+Here are instructions for different scenarios:
+_Note: Make sure you're running these commands in the root of telescope project. If any of the commands below are failing, use the command `pwd` to find your current directory and navigate back to project root (e.g., `cd <the path of directory you place telescope project>/telescope`)_
 
-#### Elasticsearch:
+#### Want to run back-end services locally, and to have front-end using the services
 
-To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
+This is the default setting, you do not need to copy or modify any `env` file.
 
-#### Login/SSO (Optional):
+```bash
+npm run services:start
 
-This step is only required if you need to login to Telescope or developing/testing requiring a logged in user. Login/SSO can be started by using the command `docker-compose up login`, this will start up a container only for Login/SSO service. For more information on Login please refer to our [Login Document](login.md).
+npm run dev
+```
 
-\_Note: Users must have docker and docker-compose installed. Please refer to `Docker and Docker-Compose Set Up` earlier in this document for more information regarding Docker.
+Back-end will start downloading feeds and processing them until stopped. The default port # for the back-end is `3000` and can be modified with the `PORT` variable in the `.env` file. Different routes such as `http://localhost:3000/posts` and
```

Comment:
Microservices instead of back-end here. Remove reference to the default ports because the services all use different ports. You can alternatively link to the README.md in the src/api directory